config.yml

```diff
@@ -13,4 +13,5 @@ channels:
     - r
     - defaults
     - conda-forge
+    - heimdali
```

Comment:
This is not allowed. But you can put the needed recipe into bioconda, conda-forge as well.